README.md

```diff
@@ -434,6 +434,22 @@ Is there an excessive amount of "premature events"? Or very old (older than a a
 
 https://help.sap.com/viewer/d0224eca81e249cb821f2cdf45a82ace/2011/en-US/7e8ff9d7653f43e8890bc8eb395d52a7.html
 
+</td></tr>
```

Comment:
This table is build automatically built based on the comments in the impexes.

- [ ] Move the query and explanation text to a comment in your impex. (check the other impex files for examples)

```
# @readme <type>
# <explanatory text in markdown>
# <QUERIES>
INSERT_UPDATE ...
```